index.js

```diff
@@ -1,4 +1,78 @@
-const answerBtn = document.querySelector(".button");
-const answerText = document.querySelector(".answer");
+// ACCESSING DATA
+// --BUTTONS
+const answerBtn1 = document.querySelector('[data-js="btn1"]');
+const answerBtn2 = document.querySelector('[data-js="btn2"]');
+const answerBtn3 = document.querySelector('[data-js="btn3"]');
+const answerText = document.querySelector('[data-js="answer1"]');
+const answerText2 = document.querySelector('[data-js="answer2"]');
+const answerText3 = document.querySelector('[data-js="answer3"]');
+// --CARDS
 const questionCard = document.querySelector(".q-card");
+const card = document.querySelector(".card");
+// --TAGLIST
+const tagList1 = document.querySelector('[data-js="tag_list1"]');
+const tagList2 = document.querySelector('[data-js="tag_list2"]');
+const tagList3 = document.querySelector('[data-js="tag_list3"]');
+// --BOOKMARK
+const bookmark1 = document.querySelector('[data-js="bookmark1"]');
+const bookmark2 = document.querySelector('[data-js="bookmark2"]');
+const bookmark3 = document.querySelector('[data-js="bookmark3"]');
+const bookmarkUnfilled = document.querySelector('[data-js="book_unfilled]');
+const bookmarkFilled = document.querySelector('[data-js="book_filled]');
```

Comment:
Apart from the missing double quotes, you need to give your filled and unfilled bookmarks unique names.
```suggestion
const bookmark1Unfilled = document.querySelector('[data-js="book1_unfilled"]');
const bookmark1Filled = document.querySelector('[data-js="book1_filled"]');
```

Reply:
oh yeah you're right - that seems to be the point! Missed that 'cause I thought a unique button would work the same way but that was surely odd! Thank you very much! :)